packages/serverless-orchestration/package.json

```diff
@@ -8,6 +8,9 @@
     "@uma/financial-templates-lib": "^1.2.0"
   },
   "devDependencies": {
+    "@google-cloud/datastore": "^6.0.0",
```

Comment:
Not having these packages previously was a false positive because they were already included by the reporter

Reply:
good catch.